src/custom/constants/index.ts

```diff
@@ -59,8 +59,8 @@ export const BUY_ETHER_TOKEN: { [chainId in ChainId]: Token } = {
 export const ORDER_ID_SHORT_LENGTH = 8
 export const INPUT_OUTPUT_EXPLANATION = 'Only executed swaps incur fees.'
 export const DEFAULT_ORDER_DELAY = 20000 // 20s
-export const EXPIRED_ORDERS_BUFFER = 45 * 1000 // 45s
-export const CHECK_EXPIRED_ORDERS_INTERVAL = 10000 // 10 sec
+export const PENDING_ORDERS_BUFFER = 60 * 1000 // 60s
```

Comment:
1. Because it's been checked using this flag https://github.com/gnosis/cowswap/blob/4db448e2280bf9459a658704f4927610ac7cbc11/src/custom/state/orders/updater.ts#L151

2. And I've changed expired to be same time delay as cancelled because both have the same problem of being finalized while a settlement is submitted, although the changes of that happening on expiration are less likely.
Made me wonder whether the `Cancelled` updater should deal with expired as well...